src/shared/utilities/childProcess.ts

```diff
@@ -64,13 +64,12 @@ export interface ChildProcessResult {
  * - call and await run to get the results (pass or fail)
  */
 export class ChildProcess {
+    private static runningProcesses: Record<string, ChildProcess> = {}
```

Comment:
Thought it'd be interesting to track this, though it looks like everything is being cleaned up correctly for every feature I've tested so far. V8's garbage collector does a good job.